hack/install-tools.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+STARTTIME=$(date +%s)
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+
+echo "Detected go version: $(go version)"
+
+go get github.com/tools/godep
```

Comment:
this repo is using glide, not godep, right?

Reply:
Yeah, removing, not needed.